lib/ddtrace/tracer.rb

```diff
@@ -160,22 +222,9 @@ def set_tags(tags)
     #   parent2 = tracer.trace('parent2')   # has no parent span
     #   parent2.finish()
     #
+    # This method accepts all the options accepted by \start_span.
     def trace(name, options = {})
-      span = Span.new(self, name, options)
-
-      # set up inheritance
-      parent = @buffer.get()
-      span.set_parent(parent)
-      @buffer.set(span)
-
-      @tags.each { |k, v| span.set_tag(k, v) } unless @tags.empty?
-
-      # sampling
-      if parent.nil?
-        @sampler.sample(span)
-      else
-        span.sampled = span.parent.sampled
-      end
+      span = start_span(name, options)
```

Comment:
Great! To be complete, at some point we'll end with (nomenclature is different but just to make it readable without "context"):
in the end we should end having 3 different ways:
* `start_active_span()`: uses context propagation and sets the span as active
* `start_manual_span()`: uses context propagation but doesn't set the active span
* `start_manual_span(ignore_active_span: true)`: doesn't use the context propagation and doesn't set the active span

For now the safest way is to use `start_span()` as a manual way, so that when we introduce these three working modes, `start_span()` will only be an alias that we can deprecate.